src/current/v23.2/cdc-queries.md

```diff
@@ -188,6 +188,57 @@ For newly inserted rows in a table, the `cdc_prev` column will emit as `NULL`.
 If you do not need to select specific columns in a table or filter rows from a changefeed, you can instead create a changefeed using the [`diff` option]({% link {{ page.version.version }}/create-changefeed.md %}#diff-opt) to emit a `before` field with each message. This field includes the value of the row before the update was applied.
 {{site.data.alerts.end}}
 
+### Reference TTL in a CDC query
```

Comment:
The changes for v23.2 appear to be the same as for v23.1, so same comments apply (or don't, at your preference)